backend/app/models/reading_list.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: reading_lists
+#
+#  id         :uuid             not null, primary key
+#  status     :string
```

Comment:
FYI that enumerize does not add a Database enum type for the `status` column. 

I still recommend adding the PSQL string enum to ensure that unknown values are not added if AR validation is bypassed.

But again - since this is mostly a training project - you can choose to ignore. 

Just bear this in mind when you start working on prod Rails apps

Reply:
updated! [here](https://github.com/ronsigter/bookshelf/pull/13/files#diff-5503774b3c7d33d8aa26d8041c1f2f4463e8380d4bd63ec499e95c254e664e4fR1) 🙆‍♂️